Dockerfile

```diff
@@ -1,23 +1,37 @@
-FROM openjdk:8-jdk
+FROM maven:3.6.3-jdk-8 as build
 
 ENV JAVA_OPTS "-server -Xconcurrentio -Xmx1g -Xms1g -XX:+UseG1GC -Ddw.server.application_connectors[0].bind_host=0.0.0.0 -Ddw.server.application_connectors[0].port=8989"
 
-RUN mkdir -p /data && mkdir -p /graphhopper
-
 # install node - only required for JS UI
 RUN apt-get install -y wget \
        && curl -sL https://deb.nodesource.com/setup_13.x | bash - \
        && apt-get install -y nodejs
 
-COPY . /graphhopper/
-
 WORKDIR /graphhopper
 
+COPY . .
+
 # create main.js - only required for JS UI
-RUN cd web && npm install && npm run bundleProduction && cd ..
+WORKDIR /graphhopper/web
+
+RUN npm install && npm run bundleProduction
+
+WORKDIR /graphhopper
 
 RUN ./graphhopper.sh build
 
+FROM openjdk:11.0.5-jre
```

Comment:
Do we need to hardcode minor/patch versions?

Reply:
Thanks for the tip. I have removed the .5, using 11.0 now.

Reply:
i think `openjdk:11-jre` should work too

Reply:
Minor version changes can still break some code, I prefer to be on the safe side. Feel free to change it though. :-)

Reply:
I prefer stable and specific versions too. So I would keep it as it is.